src/modals/SecretPhraseModal.tsx

```diff
@@ -33,23 +33,23 @@ const SecretPhraseModal = ({ onClose }: { onClose: () => void }) => {
   const [isDisplayingPhrase, setIsDisplayingPhrase] = useState(false)
 
   return (
-    <CenteredModal title={t`Secret phrase`} onClose={onClose} focusMode narrow={!isDisplayingPhrase}>
+    <CenteredModal title={t`Recovery phrase`} onClose={onClose} focusMode narrow={!isDisplayingPhrase}>
```

Comment:
I'd add the `secret` word there so that it's mentioned at least once in this modal:

<img width="632" alt="image" src="https://user-images.githubusercontent.com/1579899/186355989-3c5604ac-46f4-444a-98ec-cb7bbbe1eead.png">